internal/handlers/api/api.go

```diff
@@ -0,0 +1,86 @@
+package api
+
+import (
+	"encoding/json"
+	"io"
+	"mmskazak/shorturl/internal/logger"
+	"mmskazak/shorturl/internal/services/genidurl"
+	"mmskazak/shorturl/internal/services/shorturlservice"
+	"net/http"
+)
+
+type IStorage interface {
```

Comment:
В го непринято, в основном, давать интрефейсам префиксы `I`

```suggestion
type Storage interface {
```

Reply:
Переименовал